common/src/IO/MapReader.cpp

```diff
@@ -287,20 +290,41 @@ namespace TrenchBroom {
             storeNode(groupNode, properties, status);
             m_groups.insert(std::make_pair(groupId, groupNode));
 
+            const auto& linkedGroupId = findProperty(properties, Model::PropertyKeys::LinkedGroupId);
```

Comment:
But `findProperty` returns a reference?

Reply:
The problem is it's called two lines further down without `&`. 

`const auto transformationStr = findProperty(properties, Model::PropertyKeys::GroupTransformation);`

This is confusing

Reply:
I would suggest using `const std::string&` for all of these local variables. Especially since there's a `findProperty` overload that returns an iterator!

```
auto it = findProperty(key); // iterator
..
const auto transformationStr = findProperty(properties, ..); // actually std::string&
```

Reply:
Sure, de00a0c16